docs/register-phase-1-stake.md

```diff
@@ -0,0 +1,154 @@
+# Register your Phase-1 stake to Phase-2 through CLI
+
+## Understanding the registration process
+
+To register your Bitcoin stake on the Babylon chain, you must have a confirmed 
+Bitcoin staking transaction. Before proceeding, please review the 
+[Registration of Existing Stakes to the Babylon Chain](https://github.com/babylonlabs-io/babylon/blob/main/docs/stake-registration.md) 
+documentation to check your eligibility and understand the registration 
+requirements and process.
+
+## Table of Contents
+- [Prerequisites](#prerequisites)
+- [Setup and configuration](#setup-and-configuration)
+- [Register your stake](#register-your-stake)
+- [Verify your registration](#verify-your-registration)
+- [Receiving rewards](#receiving-rewards)
+
+## Prerequisites
+
+To register your stake to the Babylon blockchain, the following are required:
+- Access to the Bitcoin wallet you have used to create your stake on Bitcoin 
+  (needed to sign transactions).
+- The Bitcoin block height in which your staking transaction was included, which 
+  will be used in the registration command.
+- A funded Babylon address, which will be used to receive the rewards from the 
+  registration.
+- Global parameter file as seen here https://github.com/babylonlabs-io/networks/blob/main/bbn-test-5/parameters/global-params.json 
+  that were used when you created your Bitcoin stake. This will be used in the 
+  registration command later on.
+- A running instance of the `stakerd` daemon. Please follow the `stakerd` 
+  [setup guide](../README.md#3-btc-staker-installation) to complete the setup. 
+  This is also inclusive of creating a Babylon keyring with funds step as the 
+  `stakerd` daemon requires a funded keyring to pay for the transactions.
+
+## Setup and configuration
```

Comment:
We don't need this section I think. I explained everything in my above recommendation on the pre-requisites.

If we need it, I think it's a bit confusing. We ask the staker as a pre-requisite to set up the stakerd daemon, but then we tell them "You need to create a new config file" and connect it to a different wallet and then start again. Doesn't make sense for me. We can just tell them (as I did in the pre-requisites edit) "set up your stakercli daemon to connect to the wallet you used". They can figure it out